velox/type/Type.h

```diff
@@ -1521,10 +1526,28 @@ inline ComplexType to(std::string value) {
   return ComplexType();
 }
 
+/// Forward declaration.
+namespace exec {
+
+class CastOperator;
+
+} // namespace exec
+
+using CastOperatorPtr = std::shared_ptr<exec::CastOperator>;
+
+class CustomTypeFactories {
```

Comment:
it would be nice to document this class and its methods.

Reply:
Added. Thanks for the suggestion!